### DISCUSSION TITLE
Only method is not present

### DISCUSSION THREAD

**Discussion by fefo-p** [livewire/livewire#6103] [7. Bugs (archived)] [answered]
### Livewire version

3.0.0@beta-5

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Just follow the Livewire.laravel documentation regarding form objects.
When trying to extract the save method into the PostForm, I get the error `Call to undefined method App\Livewire\Forms\PostForm::only()` because the class `Livewire\Features\SupportFormObjects\Form::class` does not have the method defined (whereas the `Livewire\Component::class` does.

```php
public function save()
    {
        Post::create(
            $this->only('title', 'content')
        );
 
        return $this->redirect('/posts');
    }
```

The `only()` method is not present in the `Livewire\Features\SupportFormObjects\Form::class`;<br>
The `all()` method works fine as it is converting the object with `toArray()` method applied to the initial collection.<br>

If I create
```php
public function only($properties): array
        {
            $array = [];
            foreach ($properties as $key) {
                $array[$key] = $this->{$key};
            }

            return $array;
        }
```
on the `PostForm::class` (created while following the documentation), everything works, but the method (however it's written) should exist in the `Livewire\Features\SupportFormObjects\Form::class`

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PerryvanderMeer** [comment, marked as answer]
It was already added in dbeff0ea3a25da692fe2e41998e07f6f77960aee and will be tagged in `beta.6`.

**Reply by fefo-p** [reply]
Thanks! Sorry for not catching it in the forum
